README.md

```diff
@@ -14,6 +14,11 @@ ember install ember-cli-blanket
 ember install:addon ember-cli-blanket
 ```
 
+## Requirements
+* If using Mocha, Testem `>= 1.6.0` for which you need ember-cli `> 2.4.3`
```

Comment:
Looks like it's related to a hook they needed from testem https://github.com/kategengler/ember-cli-code-coverage/commit/c132f844b59c60e4b51e4071e131e9113efeda36#diff-04c6e90faac2675aa89e2176d2eec7d8

I don't think we have the same requirement - the mirage and pretender is definitely relevant though